### DISCUSSION TITLE
Trying to run Authelia and npm in one configuration

### DISCUSSION THREAD

**Discussion by zeusraman** [authelia/authelia#6210] [Q&A] [answered]
hi all,
 followed https://jnsn.me/blog/use-authelia-with-nginx-proxy-manager/
Error:
uthelia    | time="2023-10-30T22:52:42+13:00" level=fatal msg="Can't continue due to the errors loading the configuration"
authelia exited with code 1
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: secrets: error loading secret into key 'jwt_secret': it's already defined in other configuration sources"
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: secrets: error loading secret into key 'session.secret': it's already defined in other configuration sources"
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: option 'jwt_secret' is required"
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: authentication_backend: you must ensure either the 'file' or 'ldap' authentication backend is configured"
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: access control: 'default_policy' option 'deny' is invalid: when no rules are specified it must be 'two_factor' or 'one_factor'"
authelia    | time="2023-10-30T22:52:46+13:00" level=error msg="Configuration: notifier: please ensure only one of the 'smtp' or 'filesystem' notifier is configured"
authelia    | time="2023-10-30T22:52:46+13:00" level=fatal msg="Can't continue due to the errors loading the configuration"
authelia exited with code 1

have created files and added 64 bit keys

can someone help ?

cheers

**Comment by james-d-elliott** [comment, marked as answer]
You must configure the authentication_backend and access_control sections, must check only the SMTP or file system notifier are configured (either by config or environment vars), and must ensure the `jwt_secret` and session `secret` are only defined in one location (config or env).
